packages/babel-parser/test/fixtures/experimental/top-level-await/inside-arrow/output.json

```diff
@@ -0,0 +1,41 @@
+{
+  "type": "File",
+  "start":0,"end":14,"loc":{"start":{"line":1,"column":0},"end":{"line":1,"column":14}},
+  "errors": [
+    "SyntaxError: 'await' is only allowed within async functions and at the top levels of modules (1:6)"
```

Comment:
It seems that `SyntaxError` automatically prepend `SyntaxError: ` before `message`.